nvflare/app_common/job_schedulers/job_scheduler.py

```diff
@@ -277,14 +277,15 @@ def _get_update_meta_keys(self):
             JobMetaKey.SCHEDULE_HISTORY.value,
         ]
 
-    def _update_schedule_history(self, job: Job, result: str):
+    def _update_schedule_history(self, job: Job, result: str, fl_ctx: FLContext):
         history = job.meta.get(JobMetaKey.SCHEDULE_HISTORY.value, None)
         if not history:
             history = []
             job.meta[JobMetaKey.SCHEDULE_HISTORY.value] = history
         now = datetime.datetime.now()
         cur_time = now.strftime("%Y-%m-%d %H:%M:%S")
         history.append(f"{cur_time}: {result}")
+        self.log_info(fl_ctx, f"Try to schedule job {job.job_id}, get result: ({result}).")
```

Comment:
It will be logged everytime we try to schedule this job.
As for how frequent, Yan or Yuhong introduced something like exponential back off in line 319

`required_interval = min(self.max_schedule_interval, (2**n) * self.min_schedule_interval)`